capstone/src/test/javascript/calendarTest.js

```diff
@@ -0,0 +1,20 @@
+describe('DataTable Construction', function() {
+  it('Will add correct columns/rows to DataTable from JSON', function() {
+    const mockCalenderInfo = {};
+    const expectedFreqData;
```

Comment:
i would think that a `const` must be initialized (which makes me think that this test isn't running...)

Reply:
Yea `consts` should be initialized upon creation ideally.